tests/integration_tests/style/test_licenses.py

```diff
@@ -105,3 +106,7 @@ def test_for_valid_licenses():
         if _validate_license(file) is False:
             error_msg.append("{}".format(str(file)))
     assert not error_msg, "Files {} have invalid licenses".format((error_msg))
+
+
+if __name__ == "__main__":
```

Comment:
Is this changed used within this PR?

Reply:
Actually no, I used it to check locally that the licenses were fine.

Reply:
Seems that it is still there